package.json

```diff
@@ -171,7 +171,7 @@
     "prosemirror-state": "^1.4.3",
     "prosemirror-tables": "^1.3.2",
     "prosemirror-transform": "^1.7.3",
-    "prosemirror-view": "^1.31.3",
+    "prosemirror-view": "^1.32.0",
```

Comment:
Get this fix: https://github.com/ProseMirror/prosemirror-view/commit/f25f340bd87b95a12ba7cd99fe6a5042b2e6920e